queries/highlights.scm

```diff
@@ -1,17 +1,28 @@
-(ERROR) @error
```

Comment:
Why remove this? Does it hurt in neovim (in helix it highlights errors, which may be helpful)?

Reply:
Oh, I'm actually not sure. Neovim doesn't highlight errors differently by default, and I'm not sure if it just automatically adds that query anyways (at least in my colorscheme `@error` isn't highlighted) because not every parser is perfect and breaking changes to syntax can be introduced quite easily and would be a pain for users to wait for a parser update. However, I don't mind adding it back

Reply:
Well it hasn't created problems for me in neovim, so I don't think it hurts.